upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -418,6 +421,34 @@ func (c *ApplyClusterCmd) Run() error {
 	l.WorkDir = c.OutDir
 	l.ModelStore = modelStore
 
+	iamLifecycle := lifecyclePointer(fi.LifecycleSync)
+	networkLifecycle := lifecyclePointer(fi.LifecycleSync)
+	clusterLifecycle := lifecyclePointer(fi.LifecycleSync)
+
+	switch c.Phase {
+	case Phase(""):
+	// Everything ... the default
+
+	case PhaseIAM:
+		networkLifecycle = lifecyclePointer(fi.LifecycleIgnore)
+		clusterLifecycle = lifecyclePointer(fi.LifecycleIgnore)
+
+	case PhaseNetwork:
+		iamLifecycle = lifecyclePointer(fi.LifecycleIgnore)
+		clusterLifecycle = lifecyclePointer(fi.LifecycleIgnore)
+
+	case PhaseCluster:
+		if c.TargetName == TargetDryRun {
+			iamLifecycle = lifecyclePointer(fi.LifecycleExistsAndWarnIfChanges)
+			networkLifecycle = lifecyclePointer(fi.LifecycleExistsAndWarnIfChanges)
+		} else {
+			iamLifecycle = lifecyclePointer(fi.LifecycleExistsAndValidates)
+			networkLifecycle = lifecyclePointer(fi.LifecycleExistsAndValidates)
```

Comment:
Can we loosen these and change them to `LifecycleExistsAndWarnIfChanges`.  Allowing this will fix multiple scenarios including not using an internet gateway.